src/Libraries/UnitsUI/UnitsUI.cs

```diff
@@ -381,4 +276,577 @@ public override IEnumerable<AssociativeNode> BuildOutputAst(List<AssociativeNode
             return new[] { AstFactory.BuildAssignment(GetAstIdentifierForOutputIndex(0), node) };
         }
     }
-}
+
+    [NodeName("Parse Unit Input")]
+    [NodeCategory(BuiltinNodeCategories.CORE_UNITS)]
+    [NodeDescription("ParseUnitInputDescription", typeof(UnitsUI.Properties.Resources))]
+    [NodeSearchTags("ParseUnitInputSearchTags", typeof(UnitsUI.Properties.Resources))]
+    [IsDesignScriptCompatible]
+    public class UnitInput : CoreNodeModels.Input.String
+    {
+        public override string NodeType
+        {
+            get
+            {
+                return "UnitInputNode";
```

Comment:
so - is this a new type for the "schema"? I think usually this would be an `Extension` node - is that correct?